app/dashboard/page.tsx

```diff
@@ -1,5 +1,7 @@
+import GeneralGridLayoutT from './_components/GeneralGridLayout'
+
 const Dashboard = () => {
-  return <div>dashboard</div>
+  return <div></div>
```

Comment:
you didn't render the grid hahah

Reply:
@ell-ska updated the general layout import syntax